GeneratorAnalysis/Scripts/GeneratorOverlay.cpp

```diff
@@ -96,6 +99,10 @@ void GeneratorOverlay() {
     PlotNames.push_back("TrueCosOpeningAngleMuonTotalProtonPlot");
     PlotNames.push_back("TrueTransverseMomentumPlot");
     PlotNames.push_back("TrueDeltaAlphaTPlot");
+    PlotNames.push_back("TrueInvariantMassPlot");
+    PlotNames.push_back("TrueInvariantMassPlot");
```

Comment:
You are pushing this plot twice (I think this might be what is causing the weird range in that one plot)